tests/postgres_tests/test_forms.py

```diff
@@ -0,0 +1,8 @@
+from . import PostgreSQLTestCase
```

Comment:
There are existing form field tests in `test_json.py` and `form_fields.py` isn't needed.